src/FeatureFlagsApiManager.php

```diff
@@ -14,9 +14,8 @@ class FeatureFlagsApiManager extends Manager
     public function createLaunchDarklyDriver(): LaunchDarklyApiProvider
     {
         $token = $this->config->get('feature-flags.providers.launchdarkly.access-token');
-        if (! is_string($token)) {
-            throw new LaunchDarklyMissingAccessTokenException();
-        }
+
+        assert(is_string($token), new LaunchDarklyMissingAccessTokenException());
```

Comment:
Just to say, `assert()` only throws in development environments. We probably want to stick with the previous way? 🤔

Reply:
Oooh ok I did not know they only throw in dev.
Too bad, I liked the syntax of it 😅

Reply:
As we remove the settings of InMemoery default, those just go away 👍